graylog2-web-interface/src/components/users/SectionComponent.jsx

```diff
@@ -2,24 +2,30 @@
 import * as React from 'react';
 import styled from 'styled-components';
 
+import Spinner from 'components/common/Spinner';
 import { Row, Col } from 'components/graylog';
 
 type Props = {
   children: React.Node,
   title: string,
+  showLoading?: boolean,
 };
 
 const Headline = styled.h2`
   margin-bottom: 15px;
 `;
 
-const SectionComponent = ({ children, title }: Props) => (
+const SectionComponent = ({ children, title, showLoading = false }: Props) => (
   <Row className="content">
     <Col xs={12}>
-      <Headline>{title}</Headline>
+      <Headline>{title}</Headline>{showLoading && <Spinner />}
```

Comment:
I like it! but we should display the spinner, like in the `UsersOverview` currently `Loading...` is being displayed under the headline.